PokeDemo/Controllers/HomeViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class HomeViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var pokemonTableView: UITableView!
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+
+    private let presenter = HomeViewPresenter(service: Service.shared)
+    private var pokemonList: [Pokemon]? = [Pokemon]()
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        pokemonTableView.register(UINib(nibName: PokemonTableViewCell.cellIdentifier(), bundle: nil), forCellReuseIdentifier: PokemonTableViewCell.cellIdentifier())
+        presenter.setViewDelegate(delegate: self)
+        presenter.getPokemonList()
```

Comment:
Since architecture is MVP. view **notifies** to presenter instead of giving **command**. Here `getPokemonList` behaves as view is giving a command to presenter to do something. Rather view must use notification sort of function names. Such as `viewLoaded` or `viewDidLoad`

```
presenter.viewDidLoad()
```